bin/fetch-configlet

```diff
@@ -1,33 +1,55 @@
-#!/bin/bash
-
-LATEST=https://github.com/exercism/configlet/releases/latest
-
-OS=$(
-case $(uname) in
-    (Darwin*)
-        echo "mac";;
-    (Linux*)
-        echo "linux";;
-    (Windows*)
-        echo "windows";;
-    (*)
-        echo "linux";;
-esac)
-
-ARCH=$(
-case $(uname -m) in
-    (*64*)
-        echo 64bit;;
-    (*686*)
-        echo 32bit;;
-    (*386*)
-        echo 32bit;;
-    (*)
-        echo 64bit;;
-esac)
-
-VERSION="$(curl --head --silent $LATEST | awk -v FS=/ '/Location:/{print $NF}' | tr -d '\r')"
-URL=https://github.com/exercism/configlet/releases/download/$VERSION/configlet-$OS-${ARCH}.tgz
-
-echo "Downloading configlet $VERSION for $OS-$ARCH..."
-curl -s --location $URL | tar xz -C bin/
+#!/usr/bin/env python3.7
+import io
+import tarfile
+import platform
+import requests
+import sys
+
+
+def download_and_extract(url):
+    resp = requests.get(url)
+    print('extracting...')
+    file = tarfile.open(fileobj=io.BytesIO(resp.content), mode='r:gz')
+    file.extractall(path='bin/')
+
+
+def get_os():
+    _os = platform.platform()
```

Comment:
Not a blocker, but the convention is `name_` to avoid shadowing, `_name` to imply "non-public".